### ISSUE TITLE
full screen on OS X 10.9 shows a bar at the top with Java2D

### ISSUE THREAD

**Issue by StanLepunK** [processing/processing#3305]
OSX 10.9 Processing 3.0a9.
When we use 
`````
void setup() {
  size(displayWidth,displayHeight) ;
}
`````
the title bar is not totally remove in Maverick mode the color of the background of computer stay. When I swith to the old display mode of OSX that's ok.
![maverick mode](https://cloud.githubusercontent.com/assets/3332269/7724155/4ceb1916-feef-11e4-856a-8c01534d407b.jpg)

**Comment by StanLepunK** [comment]
That's happen, when we change the parameter.
I start with
```
void setup() {
  size(displayWidth,300) ;
}
````
next I change to
void setup() {
  size(displayWidth,displayHeight) ;
}
````
It's like Processing keep the old position in memory.
When I close Processing and Open again Processing the problem dissapear.

**Comment by benfry** [comment]
Not seeing this in 10.10.

**Comment by StanLepunK** [comment]
The best definition, "full screen on OSX 10.9 keep the window position when you change the size parameter with JAVA2D". 
that's happen with the title when I test with 
````
void setup() {
  //size(300,displayHeight) ;
  size(displayWidth,displayHeight) ;
}
`````

**Comment by benfry** [comment]
Should be fixed for 3.0 alpha 10.
